types/resize-observer-browser/test/resize-observer-browser-module.tests.ts

```diff
@@ -1,5 +1,3 @@
-import { ResizeObserver } from "resize-observer-browser";
```

Comment:
It looks like we don't need a module test at all, given there is no import here.
Let's rename this file to `resize-observer-browser-tests.ts` (as per https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package), and remove the global tests file.

Then we should merge in the `window.ResizeObserver` test into here.

Reply:
Oh, I think the file should be moved out of the `tests/` subdirectory as well if we do this.

Reply:
Done.